src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+
+  if (days >= 7) {
+    return days * price - 50;
```

Comment:
![image](https://github.com/user-attachments/assets/92fcadde-e306-48b4-90ce-5c023a660b83)
Also, let's create a variable for `days * price` to avoid duplication